query-connector/src/app/backend/dbServices/db-service.ts

```diff
@@ -0,0 +1,14 @@
+import { Pool } from "pg";
+import { getDbClient } from "../dbClient";
+import { camelCaseDbColumnNames } from "./decorators";
+
+class dbService {
```

Comment:
would love some extra thoughts on this declaration / implementation in `audit-logs.ts`! Doing this to be able to centralize the camel casing logic in our DB queries and any future db-level functionality we might want to layer on.

If folks think this is good, happy to move over the code from the other file / put up a PR to refactor the other DB services accordingly.

Reply:
Yeah, I think it's a good idea to use this globally, but definitely save it for another PR!

Reply:
https://linear.app/skylight-cdc/issue/QUE-240/use-camel-case-decorator-everywhere